apps/explorer/src/pages/index.tsx

```diff
@@ -38,6 +39,7 @@ export const router = sentryCreateBrowserRouter([
 			{ path: 'address/:id', element: <AddressResult /> },
 			{ path: 'validators', element: <ValidatorPageResult /> },
 			{ path: 'validator/:id', element: <ValidatorDetails /> },
+			{ path: 'object-address/:id', element: <ObjectAddress /> },
```

Comment:
Since we can't feature flag the route can we do something like `experimental--object-address/:id` just so it's a little more obvious?

Reply:
Updated